thesis-gui/src/services/quizzes-service.ts

```diff
@@ -8,5 +8,17 @@ export const quizzesService = {
 
     createNew(questionsSet: QuestionsSetCreateRequest) {
         return httpService.post<QuestionsSetCreateRequest, QuestionsSetKeyResponse>('/questions-sets', questionsSet);
+    },
+
+    getQuiz(key: string) {
+        return httpService.get<QuizFullResponse>(`/questions-sets/${key}`);
+    },
+
+    update(key: string, questionsSet: QuestionsSetCreateRequest) {
+        return httpService.put<QuestionsSetCreateRequest, QuestionsSetKeyResponse>(`/questions-sets/${key}`, questionsSet);
```

Comment:
I would change this `QuestionsSetCreateRequest` type to `QuestionsSetRequest` or sth like that as now it's used in `createNew` as well as `update` methods

Reply:
Done.